audb/core/publish.py

```diff
@@ -262,18 +356,28 @@ def publish(
     r"""Publish database.
 
     A database can have dependencies
-    to files of an older version of itself.
-    E.g. you might add a few new files to an existing database
-    and publish as a new version.
-    :func:`audb.publish` will upload then only the new files
-    and store dependencies on the already published files.
-
-    To allow for dependencies
-    you first have to load the version of the database
+    to media files and tables of an older version.
+    E.g. you might alter an existing table
+    by adding new media files to it
```

Comment:
To be more precise maybe:
```suggestion
    by adding labels for new media files to it
```